docs/JestObjectAPI.md

```diff
@@ -364,6 +360,16 @@ Executes only the macro-tasks that are currently pending (i.e., only the tasks t
 
 This is useful for scenarios such as one where the module being tested schedules a `setTimeout()` whose callback schedules another `setTimeout()` recursively (meaning the scheduling never stops). In these scenarios, it's useful to be able to run forward in time by a single step at a time.
 
+### `.jest.setSystemTime()`
+
+Set the current system time used by fake timers, affecting the time returned by `ned Date` et.al.
+
+See the [Lolex docs](https://github.com/sinonjs/lolex/blob/master/Readme.md#clocksetsystemtimenow) for more information.
```

Comment:
I think we could use the wording from the current Lolex docs instead of (or next to) linking to it.

Reply:
Shortened our part, kept the link. thoughts?

Reply:
I think I'd rather like:

>Set the current system time used by fake timers. Simulates a user changing the system clock while your program is running. It affects the current time but it does not in itself cause e.g. timers to fire; they will fire exactly as they would have done without the call to `jest.setSystemTime()`.

Really no need to link to such a short explanation. However, somewhere at the top of Fake Timer docs, we could state that we use Lolex and here's the docs to it if you're curious, but you shouldn't :p

Reply:
Went for that.

We really should group information in that document - one section for module mocks, one for functions pies, one for timers etc. Now it's all jumbled up

Reply:
Agreed. @rickhanlonii ideas on making that happen?

Reply:
#7278